notify_test.go

```diff
@@ -59,7 +59,7 @@ func TestNotifieeFuzz(t *testing.T) {
 	d1 := setupDHT(ctx, t, false)
 	d2 := setupDHT(ctx, t, false)
 
-	for i := 0; i < 100; i++ {
+	for i := 0; i < 10; i++ {
```

Comment:
@anacrolix the description of the PR implies we're _adding_ a test, but we're actually modifying an existing one. What was the problem here?